specs/lib/updateMaterialPropertiesSpec.js

```diff
@@ -0,0 +1,236 @@
+'use strict';
+var Cesium = require('cesium');
+var updateMaterialProperties = require('../../lib/updateMaterialProperties');
+
+var WebGLConstants = Cesium.WebGLConstants;
+
+describe('updateMaterialProperties', function() {
+    it('sets material.doubleSided property if CULL_FACE is not enabled', function () {
+        var gltf = {
+            programs: {
+                program_0: {
+                    attributes: [
+                        'a_normal',
+                        'a_position',
+                        'a_texcoord0'
+                    ],
+                    fragmentShader: 'BoxTextured0FS',
+                    vertexShader: 'BoxTextured0VS'
+                }
+            },
+            shaders: {
+                BoxTextured0FS: {
+                    type: WebGLConstants.FRAGMENT_SHADER,
+                    uri: 'BoxTextured0FS.glsl'
+                },
+                BoxTextured0VS: {
+                    type: WebGLConstants.VERTEX_SHADER,
+                    uri: 'BoxTextured0VS.glsl'
+                }
+            },
+            techniques: {
+                technique0: {
+                    states: {
+                        enable: [
+                            WebGLConstants.DEPTH_TEST,
+                            WebGLConstants.BLEND
+                        ]
+                    }
+                }
+            },
+            materials: [
+                {
+                    technique: 'technique0'
+                }
+            ],
+            extensionsUsed: [],
+            extensionsRequired: []
+        };
+
+        var gltfWithUpdatedMaterials = updateMaterialProperties(gltf);
+        var material = gltfWithUpdatedMaterials.materials[0];
+        expect(material.doubleSided).toBe(true);
+
+        gltf = {
+            programs: {
+                program_0: {
```

Comment:
To make this more condensed clone the previous gltf and add the new state.

This can probably be done to make all the tests shorter. Have a single glTF above the tests that is cloned/edited by each test.